### PR TITLE
Cleans up unnecessary transformation options in admin player panel

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
I got tired of having to walk new admin candidates through the admin buttons and telling them about all of the antiquated buttons they'd never use, so I'm doing my part and removing a bunch of the transformation bloat from the player panel. Below is the current menu

[](https://i.imgur.com/etAwhza.png)

The only button admins I know ever actually use is the human rudimentary transformation one, and literally nothing else. Judging by how the options are a random choice of some of the station pets, these options probably haven't been updated since like 2012. In addition, the non-rudimentary transformations only work if you're a human, so not sure why they show up for non human player panels.

[](https://i.imgur.com/pHvhWPK.png)

Here's the new pared down selection, I figured these would be the only useful ones from the bunch. They work no matter what mob the target was previously, whether it be another living mob or a ghost. In addition, I also cut out the transformation equivalents from the VV dropdown, because lord knows that menu keeps getting longer and longer. Lastly, I refactored stuff where possible without getting too trapped in cursed pre-MSO era admincode.

## Why It's Good For The Game
Removes a bunch of unused bloat from one of the most commonly used admin panels, making the admin tools a bit less unintuitive

## Changelog

:cl: Ryll/Shaps
admin: Simplified the "Transformation/Rudimentary Transformation" part of the admin player panel, you can now choose from ghost/human/monkey/cyborg/AI, and all options work whether the target is currently a human/other living mob/ghost.
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
great pr!

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -773,9 +689,9 @@
 		if(!check_rights(R_SPAWN))
 			return
 
-		var/mob/living/carbon/human/H = locate(href_list["makeai"])
-		if(!istype(H))
-			to_chat(usr, "This can only be used on instances of type /mob/living/carbon/human.", confidential = TRUE)
+		var/mob/our_mob = locate(href_list["makeai"])
```
Comment (Watermelon914) [inline code/modules/admin/topic.dm:692]: make sure to verify that this is actually a mob. Admins might press the button on a delete mob and it'll cause a runtime

**Review by Watermelon914 — CHANGES_REQUESTED** [review]

**Comment by Fikou** [comment]
while youre touching player panel, subtle message and traitor panel both appear in it twice

**Comment by Ryll-Ryll** [comment]
i'm too indecisive to decide which SM/TP to delete ;_;

**Comment by Bawhoppen** [comment]
I'm sad this is gone

**Pull request merged**